src/wasm2js.h

```diff
@@ -2240,11 +2270,33 @@ void Wasm2JSGlue::emitPre() {
     emitPreES6();
   }
 
+  if (isTableExported(wasm)) {
+    out << "function FakeTable(size) {\n"
```

Comment:
I think just `Table` is a good enough name? `"Fake"` doesn't clarify things to me at least.

Reply:
I was hoping to convey some kind meaning around how it mimics the API of the WebAssembly::Table?  How about ExportedTable?

Reply:
I think `Table` is clearer since it's a polyfill for `WebAssembly.Table`?